lib/beaker/host/cisco.rb

```diff
@@ -72,7 +72,7 @@ def prepend_commands(command = '', user_pc = '', opts = {})
       return user_pc unless command.index('vsh').nil?
 
       prepend_cmds = 'source /etc/profile;'
-      if self[:platform] =~ /cisco-5/
+      if self[:platform] =~ /cisco-nexus/
```

Comment:
This should probably instead be `if self[:vrf]` - use the VRF if defined, don't use it if not.